libs/db-browser/src/main/java/com/oceanbase/tools/dbbrowser/util/TimestampUtils.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2023 OceanBase.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.oceanbase.tools.dbbrowser.util;
+
+import java.sql.Timestamp;
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.TimeZone;
+
+import javax.validation.constraints.NotBlank;
+
+import lombok.NonNull;
+import lombok.extern.slf4j.Slf4j;
+
+/**
+ * @author jingtian
+ * @date 2024/7/8
+ */
+@Slf4j
+public class TimestampUtils {
+    public static Timestamp convertToDefaultTimeZone(@NonNull Timestamp timestamp, @NotBlank String timeZone) {
+        DateFormat sdf = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss");
```

Comment:
unnessesary format and parse may involve performance cost,
as timestamp is really an long type, we may use value based convert straightly.

e.g. convert timestamp to instant long value. 

and, I try to understand  TimeZone.getDefault(), is it hardcode as UTC+8, sed in Dockerfile ?